ext/openssl/openssl.c

```diff
@@ -4933,6 +4941,44 @@ PHP_FUNCTION(openssl_dh_compute_key)
 }
 /* }}} */
 
+/* {{{ proto string openssl_derive(resource peer_pub_key, resource priv_key,int keylen=NULL)
+   Computes shared secret for public value of remote and local DH or ECDH key */
+PHP_FUNCTION(openssl_derive)
+{
+	zval *priv_key;
+	zval *peer_pub_key;
+	EVP_PKEY *pkey;
+	EVP_PKEY *peer_key;
+	zend_long keylen = 0;
+	zend_string *result;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "rr|l", &peer_pub_key, &priv_key, &keylen) == FAILURE) {
+		return;
+	}
+	if ((pkey = (EVP_PKEY *)zend_fetch_resource(Z_RES_P(priv_key), "OpenSSL key", le_key)) == NULL
+	 || (peer_key = (EVP_PKEY *)zend_fetch_resource(Z_RES_P(peer_pub_key), "OpenSSL key", le_key)) == NULL) {
+		RETURN_FALSE;
+	}
+	EVP_PKEY_CTX *ctx = EVP_PKEY_CTX_new(pkey, NULL);
+	if (!ctx) RETURN_FALSE;
+	if (EVP_PKEY_derive_init(ctx) > 0
+	 && EVP_PKEY_derive_set_peer(ctx, peer_key) > 0
```

Comment:
NIT CS: usually tab should used for indent of `&&` on the next line
```
if (EVP_PKEY_derive_init(ctx) > 0
		&& EVP_PKEY_derive_set_peer(ctx, peer_key) > 0
```

Reply:
@bukka  Thanks for the feedback!

As for padding for a wrapped line - wanted to make it distinct from the padding on the following block,
changed to tabs now as it's consistent with other places in the code